massgov/mayflower
# Patternlab/DP-19336 offscreen menu fixes

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description


## Related Issue / Ticket

- [JIRA issue]()
- [Github issue]()

## Steps to Test


1. Test changes in Mayflower on the hamburger menu mixed page, which should now have emergency alerts.
2. Switch to this branch on openmass: `DP-19120-global-nav-drupal-changes` and run this command to bring MF changes in:`composer require massgov/mayflower-artifacts:dev-patternlab/DP-19336-offscreen-menu-fixes`
3. On both openmass and mayflower, check that you can scroll to see the entire menu even when the alerts are in full view.
4. Also, check when alerts are not present.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* Header
* Main Navigation (hamburger, non-homepage)

#### @TODO


*

#### Today I learned...
